docs/install.md

```diff
@@ -3,120 +3,307 @@ SPDX-FileCopyrightText: The RamenDR authors
 SPDX-License-Identifier: Apache-2.0
 -->
 
-# Install
+# Installing Ramen
+
+This guide describes how to install Ramen's disaster recovery operators on your
+cluster environment.
+
+## Overview
+
+Ramen consists of two operators that work together to provide disaster recovery
+capabilities:
+
+1. **Ramen Hub Operator** - Installed on the OCM hub cluster, orchestrates DR
+   operations across managed clusters
+1. **Ramen DR Cluster Operator** - Installed on each OCM managed cluster,
+   manages local volume replication and workload protection
 
 ## Prerequisites
 
-### OCM managed multi-cluster setup
+Before installing Ramen, ensure the following requirements are met:
+
+### 1. Kubernetes Versions
+
+**Supported versions:**
+
+- Kubernetes 1.20 or higher
+
+### 2. Open Cluster Management (OCM) Setup
+
+Ramen requires an
+[OCM](https://open-cluster-management.io/docs/concepts/architecture/) hub cluster
+with at least two managed clusters for disaster recovery operations.
+
+**Requirements:**
+
+- OCM hub cluster with `multicluster-engine` installed
```

Comment:
I am not sure about "multicluster-engine" installation.

We have a few addons that we require, as we install in drenv, and we should leverage that as an example here.

See:
- https://github.com/RamenDR/ramen/blob/a5256babf81df88ef62014f6e22fc1f49cb0fc90/test/addons/ocm-hub/start#L19-L28
- https://github.com/RamenDR/ramen/blob/a5256babf81df88ef62014f6e22fc1f49cb0fc90/test/addons/ocm-controller/kustomization.yaml#L11
- (for managed cluster) https://github.com/RamenDR/ramen/blob/a5256babf81df88ef62014f6e22fc1f49cb0fc90/test/addons/ocm-cluster/start#L22-L35